views/help_menu/git_hub_message.txt.erb

```diff
@@ -0,0 +1,4 @@
+GitHub is home to all kinds of software projects, from simple programs to today’s most popular apps.
```

Comment:
Could you add some text that explains github's ability to track issues and manage projects? 

Can you also add a little blurb about github being a place where git repositories live to reinforce that git and github are different things?